presto-influx/src/test/java/io/prestosql/plugin/influx/TestInfluxQL.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.prestosql.plugin.influx;
+
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.assertEquals;
+
+public class TestInfluxQL
+{
+    @Test
+    public void test()
+    {
+        InfluxQL test = new InfluxQL();
+        test.addIdentifier("hello").append(" = ").add("world");
+        assertEquals("hello = 'world'", test.toString());
```

Comment:
The right order is `assertEquals(actual, expected);`

Reply:
Arggh, what a silly inconsistency between testng and junit

https://javadoc.io/doc/org.testng/testng/6.8.17/org/testng/Assert.html vs https://javadoc.io/doc/org.testng/testng/6.8.17/org/testng/Assert.html